plugins_/lib/fileconv/loaders.py

```diff
@@ -172,11 +174,17 @@ def __init__(self, window, view, file_path=None, output=None, *args, **kwargs):
 
         path = os.path.split(self.file_path)[0]
         if isinstance(output, OutputPanel):
-            output.set_path(path, self.file_regex)
             self.output = output
         else:
-            self.output = OutputPanel(self.window, self.output_panel_name,
-                                      file_regex=self.file_regex, path=path)
+            self.output = OutputPanel.create(
+                self.window,
+                self.output_panel_name
+            )
+
+        SettingsDict(self.output.view.settings()).update(
```

Comment:
This seems like overkill. Not really necessary to add a dependency compared to fetching the settings object and assigning the settings normally. It would even be less lines.

Reply:
Done.